src/index.js

```diff
@@ -1,53 +1,70 @@
 'use strict'
 
-var invert = require('invert-hash')
+const bs58 = require('bs58')
 
-var mh = module.exports = function () {
-  if (arguments.length === 1) {
-    return mh.decode.apply(this, arguments)
-  } else if (arguments.length > 1) {
-    return mh.encode.apply(this, arguments)
+const cs = require('./constants')
+
+exports.toHexString = function toHexString (m) {
+  if (!Buffer.isBuffer(m)) {
+    throw new Error('must be passed a buffer')
+  }
+
+  return m.toString('hex')
+}
+
+exports.fromHexString = function fromHexString (s) {
+  return new Buffer(s, 'hex')
+}
+
+exports.toB58String = function toB58String (m) {
```

Comment:
that would make these very long, e.g. 
```js
multihash.convertToB58String(myBuffer)
// compared to
multihash.toB58String(myBuffer)
```
Because you call it on the module I fell it's pretty clear that you convert from multihash to<thing>